.github/ISSUE_TEMPLATE/bug-issue.md

```diff
@@ -0,0 +1,24 @@
+## What happened?
+_Describe in detail what went wrong; screenshots, videos, or gifs are strongly encouraged_
+
+## Specs:
+- Device: (e.g. iPhone 8, Macbook)
+- Browser: (e.g. Firefox, IE 11)
+- Test User _(if applicable)_:
+
+## Steps to Reproduce
+- URL:
+- Test User _(if applicable)_:
+
+## Desired behavior
+_Describe in detail what was supposed to happen; reference existing Github issue that contains a solution if possible_
+
+## Acceptance Criteria
+- [ ] _What change needs to be made?_
+
+## Definition of Done
+- [ ] **Attached to a Milestone** (when will this be completed?)
+- [ ] **Attached to Epic** (what body of work is this a part of? possibly `Ongoing Maintenance`)
+- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `triage`, `tools-improvements`)
+- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `call center`, `research`, `accessibility`, `content`)
+- [ ] **Labeled with `Bug`**
```

Comment:
can you put this at the top? (here and the other templates)


```
---
labels: bug
---
```

https://help.github.com/en/articles/manually-creating-a-single-issue-template-for-your-repository

Reply:
I just did this for the `bug` issue. Zenhub automatically tags Epic tickets with the `Epic` label so we should be good there. As far as the standard issue template, I don't think we want to add any front matter there cause they presumably will all be different